nx.json

```diff
@@ -0,0 +1,55 @@
+{
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "nx/tasks-runners/default",
+      "options": {
+        "cacheableOperations": []
+      }
+    }
+  },
+  "targetDefaults": {
+    "build": {
+      "dependsOn": [
+        "^build"
+      ]
+    },
```

Comment:
Dependencies are only enforced when running scripts using Nx. Doing `npm run build -w @coveo/quantic` behaves the same now as it did before.

If you want to build all projects, you would use `npx nx run-many --target=build`. If you want to build one project and all its parents, you would use `npx nx run --project=quantic --target=build`.